site-src/contributing/contributor-ladder.md

```diff
@@ -0,0 +1,132 @@
+* Encourage new contributors to work towards formal roles and responsibilities
+  in the project
+* Clearly define the path towards leadership roles
+* Develop a strong leadership pipeline so we have great candidates to fill
+  project leadership roles
+
+
+## Scope
+
+The following repositories are covered by this doc:
+
+* [kubernetes-sigs/gateway-api](https://github.com/kubernetes-sigs/gateway-api)
+* [kubernetes-sigs/ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway)
+* kubernetes-sigs/blixt (once migration is complete)
+
+Within each of these projects, there are opportunities to become an approver or
+reviewer for either the entire project, or a subset of that project. For
+example, you could become a reviewer or approver focused on just docs, GEPs, API
+changes, or conformance tests.
+
+## General Guidelines
+
+### 1. Everyone is welcome
+
+We appreciate all contributions. You don’t need to have a formal role in the project to make or review pull requests, and help with issues or discussions. Accepting a formal role within the project is entirely optional.
+
+### 2. These roles require continued contributions
+
+Applying for one of the roles defined above should only be done if you intend to
+continue to contribute at a level that would merit that role. If for any reason
+you are unable to continue in one of the roles above, please resign. Members
+with an extended period away from the project with no activity will be removed
+from the Kubernetes GitHub Organizations and will be required to go through the
+org membership process again after re-familiarizing themselves with the current
+state.
+
+### 3. Don’t merge without consensus
+
+If you have reason to believe that a change may be contentious, please wait for
+additional perspectives from others before merging any PRs. Even if you have
+access to merge a PR, it doesn’t mean you should. Although we can’t have PRs
+blocked indefinitely, we need to make sure everyone has had a chance to present
+their perspective.
+
+### 4. Start a discussion
+
+If you’re interested in working towards one of these roles, please reach out to
+a Gateway API maintainer on Slack.
+
+## Contributor Ladder
+
+The Gateway API contributor ladder has the following steps:
+
+1. Member
+2. Reviewer
+3. Approver
+4. Maintainer
+
+This is also a GAMMA-specific leadership role that does not fit as cleanly on
+this ladder. All of these roles will be defined in more detail below.
+
+## Member, Reviewer, and Approver
+
+The first steps on the contributor ladder are already [clearly defined in the
+upstream Kubernetes
+Community](https://github.com/kubernetes/community/blob/master/community-membership.md#community-membership).
+Gateway API follows those guidelines along with the rest of the Kubernetes
+community. Within Gateway API, there are a variety of areas one can become a
+reviewer or approver, this includes:
+
+* Conformance
+* Documentation
+* GEPs
```

Comment:
Can `Webhook Validation` be one of these areas? IMO, it also has code to be reviewed.

Reply:
Good call, added, thanks!